readme.md

```diff
@@ -81,7 +98,7 @@ Your Rails database has two tables. `students` has the columns `id` and `name`,
 Use ActiveRecord to create a new `pbj` sandwich and make it belong to the student named Geraldo.
 
 ```rb
-# Your answer...
+Sandwich.create(id:"", type: "pbj", student_id: "Geraldo")
```

Comment:
Close, you could also do it like:
Student.find_by(name: "Geraldo").sandwiches.create(type: "pbj")
# or
@sammich = Sandwich.create(type: "pbj")
@student = Student.find_by(name: "Geraldo")
@sammich.update(student: @student)